src/users/UserList.jsx

```diff
@@ -135,9 +135,19 @@ export default class UserList extends React.Component {
 
     const {filters} = this.props;
     var userListContent = this.props.users.length ? this.getUserList(this.props.users) : noUsersMessage;
-    var sortableFilters = filters.filterList.map(filter => {
-      return {label: filters[filter].description, value: filter};
-    });
+
+    // get a list of only the sortable filters
+    var sortableFilters = [];
```

Comment:
`for(var i in ...)` was made for objects with string keys, not arrays, it can end up enumerating other properties.

How about this approach?

```js
var sortableFilters = filters.filterList
  .filter(filter => filter.sortable)
  .map(filter => ({ label: filters[filter].description, value: filter }));
```